Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -226,6 +226,15 @@ for this property.
 - (void)_frames:(void (^)(_WKFrameTreeNode *))completionHandler WK_API_AVAILABLE(macos(11.0), ios(14.0));
 - (void)_frameTrees:(void (^)(NSSet<_WKFrameTreeNode *> *))completionHandler WK_API_AVAILABLE(macos(14.0), ios(17.0));
 
+/*! @constant WKWebViewSessionStateTypeSessionStorage Session Storage. */
+WK_EXTERN NSString * const WKWebViewSessionStateTypeSessionStorage WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
having constants in new SPI/API is generally discouraged I think. Can this be an NS_ENUM instead?

Reply:
I believe we can't have an NSSet of NS_ENUMs.

Reply:
Why don't we simply reuse `WKWebsiteDataTypeSessionStorage` like in the WKWebsiteDataStore API?

Reply:
@rr-codes I tried NS_ENUM and NS_OPTIONS but it seems neither can be held inside a NSSet. I see other APIs using NSString so that's what I have here at the moment. Do you have any other ideas of what to use or is NSString the way to go?